### DISCUSSION TITLE
Can I remove 15min limitation of Hindsight?

### DISCUSSION THREAD

**Discussion by gmooklee** [gopro/labs#279] [Q&A]
Hindsight turns itself off automatically if you don't use the Gopro for 15 min. This is very frustrating. I am using it with Volta(there is a lot to complain about this battery pack), and I am getting a larger battery pack. I even have a plan to use multiple battery packs in parallel connected. So the battery life is not an issue for me. Is this doable with the Gopro labs feature? Please help.

**Comment by gmooklee** [comment]
I found this command from the GoPro firmware update note. 
!MHSTO=x

How do I put it in the boot command like this?
!MBOOT="!Lboot"!SAVEboot=hS1

Should I do this?
!MBOOT="!Lboot"!SAVEboot=!MHSTO=360

Also, should I put the hS1 together? I still need this feature, too.

**Reply by dnewman-gpsw** [reply]
Anything starting with !M doesn't need to go into a boot command, and !M is for permanent storage.  Just scan !MHSTO=60, and hindsight will run for 60 minutes.  

You boot command adding hS1 is fine as it is.

**Reply by gmooklee** [reply]
Thank you, sir. Can I put a larger number? 360 or even larger.

**Reply by dnewman-gpsw** [reply]
sure, nothing is thoroughly tested, but they aren't any limitations.

**Reply by gmooklee** [reply]
I just tried it, and it overheated(5.3k/30p, GoPro 10, iso 100, stationary) after 20 minutes. I removed the internal battery and plugged it into external power, and now Hindsight is on for two hours. Thank you again, have a wonderful day, sir.
